packages/common/src/components/Preview/index.tsx

```diff
@@ -507,6 +512,14 @@ class BasePreview extends React.Component<Props, State> {
     });
   };
 
+  handleHashedURL = url => {
```

Comment:
Would be better if this name would be a bit more descriptive, maybe something like `refreshHashedUrl`.

Reply:
Done